docs/FAQ.md

```diff
@@ -0,0 +1,106 @@
+
+**Section 1: [Performance](#performance)**
+* [Performance Troubleshooting Checklist](#performance-troubleshooting-checklist)
+* [What type of GPU is supported?](#What-type-of-GPU-is-supported)
+* [Which Execution Provider will give me the best performance?](#Which-Execution-Provider-will-give-me-the-best-performance)
+* [Do the GPU builds support quantized models?](#Do-the-GPU-builds-support-quantized-models)
+* [Why is the GPU build slower than CPU?](#Why-is-the-GPU-build-slower-than-CPU)
+* [My converted Tensorflow model is so slow - why?](#My-converted-Tensorflow-model-is-so-slow---why)
+* [Does the Python API add any overhead to performance?](#Does-the-Python-API-add-any-overhead-to-performance)
+* [Why is ORT slower than PyTorch when using GPU?](#Why-is-ORT-slower-than-PyTorch-when-using-GPU)
+
+
+**Section 2: [Other Questions and Issues](#other-questions-and-issues)**
+* [I can't build the [xyz] execution provider.](#I-cant-build-the-xyz-execution-provider)
+* [How do I turn on verbose mode logging in Python?](#How-do-I-turn-on-verbose-mode-logging-in-Python)
+* [How do I infer models that have multiple inputs and outputs using the C/C++ API?](#How-do-I-infer-models-that-have-multiple-inputs-and-outputs-using-the-CC-API)
+* [How do I force single threaded execution mode in ORT?](#How-do-I-force-single-threaded-execution-mode-in-ORT)
+
+***
+# Performance 
+
+## Performance Troubleshooting Checklist
+Here is a list of things to check through when assessing performance issues.
+* Have you read through the [Performance Tuning guide](./ONNX_Runtime_Perf_Tuning.md)?
+* Are you using OpenMP? OpenMP will parallelize some of the code for potential performance improvements. This is not recommended for running on single threads.
+* Have you enabled all [graph optimizations](./ONNX_Runtime_Graph_Optimizations.md)? The official published packages do enable all by default, but when building from source, check that these are enabled in your build.
+* Have you searched through prior filed [Github issues](https://github.com/microsoft/onnxruntime/issues) to see if your problem has been discussed previously? Please do this before filing new issues.
+* If using CUDA or TensorRT, do you have the right versions of the dependent libraries installed? 
+
+## What type of GPU is supported? 
+The default ONNX Runtime GPU execution provider is for NVIDIA GPUs with CUDA. This build is published on Nuget: [Microsoft.ML.Onnxruntime.Gpu](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.gpu) and PyPi: [onnxruntime-gpu](https://pypi.org/project/onnxruntime-gpu). To build from source, see [BUILD.md](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#cuda). 
+
+Additionally, for NVIDIA GPUs you can build the [TensorRT execution provider](./execution_providers/TensorRT-ExecutionProvider.md) following [these instructions from BUILD.md](./../BUILD.md#tensorrt)
```

Comment:
let's add a section for Nvidia ARM support. (L4T , Linux for Tegra)
Will write up something and send offline.

Reply:
I will add something in a separate PR. the list of steps needs to be formally documented.
currently there's a comment in a github issue which documents the steps.